Zuri-Portfolio-Explore/Domains/DTOs/Request/PortfolioDTO.cs

```diff
@@ -8,15 +11,39 @@ public class PortfolioDTO
         public List<string> Skills { get; set; }
         public List<ProjectDTO> Projects { get; set; }
     }
+    /// <summary>
+    /// Specifies the sorting options.
+    /// </summary>
+    public enum SortBy
+    {
+        /// <summary>
+        /// Sort by newest first.
+        /// </summary>
+        [Display(Name = "Newest")]
+        [EnumMember(Value = "newest")]
+        Newest = 1,
+
+        /// <summary>
+        /// Sort by oldest first.
+        /// </summary>
+        [Display(Name = "Oldest")]
+        [EnumMember(Value = "oldest")]
+        Oldest = 2
+    }
     public class PortfolioFilterDTO
     {
+        public SortBy SortBy { get; set; } = SortBy.Newest;
         public string? Location { get; set; }
+        public DateTime? CreatedAtMin { get; set; }
+        public DateTime? CreatedAtMax { get; set; }
```

Comment:
why do we have these two different properties

Reply:
Users can now filter portfolios based on the creation date using CreatedAtMin and CreatedAtMax parameters, allowing them to retrieve portfolios created within a specific date range.